### PR TITLE
1040 update PR template, add sandbox and more

### PR DESCRIPTION
**Pull request by leite08**
Ref: metriport/metriport-internal#1040

### Dependencies

none

### Description

Update PR template, add sandbox and more

### Testing

none

### Release Plan

nothing special

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -7,20 +7,38 @@ Ticket: #_[ticket-number]_
 
 ### Description
 
+_[Regular PRs:]_
```
Comment (Goncharo) [inline .github/pull_request_template.md:10]: nit: the other titles don't have a `:`, let's be consistent

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -7,20 +7,38 @@ Ticket: #_[ticket-number]_
...
+
 _[Document your changes, give context for reviewers, add images/videos of UI changes]_
 
+_[Release PRs]_
+
+PSs:
```
Comment (Goncharo) [inline .github/pull_request_template.md:16]: nit: typo, and is this line needed in any case?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -36,7 +52,7 @@ _[Add and remove items below accordingly]_
   - [ ] _step2_
 - [ ] Added to [monthly product update](https://www.notion.so/metriport/Customer-Updates-21b4e9d3ad5f4fd68db587a11db28cff?pvs=4) (deprecates a feature that needs to be communicated with customers)
 - [ ] Added to [monthly product update](https://www.notion.so/metriport/Customer-Updates-21b4e9d3ad5f4fd68db587a11db28cff?pvs=4) (introduce a feature that would be useful customers)
-- [ ] Upstream dependencies are met
+- [ ] Upstream dependencies are met/released
 - [ ] Release NPM packages
```
Comment (jonahkaye) [inline .github/pull_request_template.md:56]: maybe add: "DB migrations if relevant"

Comment (leite08) [reply .github/pull_request_template.md:56]: There's already an item about DB migrations, @jonahkaye . Do you mean to add "if relevant" there?

If so, then we would need to add to every item, and there's already a note indicating to add/remove/edit the items as needed.

Comment (leite08) [reply .github/pull_request_template.md:56]: @jonahkaye bump ^

**Review by RamilGaripov — APPROVED** [review]
lgtm

**Pull request merged**